packages/ckeditor5-ckbox/src/ckboxediting.ts

```diff
@@ -62,8 +63,19 @@ export default class CKBoxEditing extends Plugin {
 	public async init(): Promise<void> {
 		const editor = this.editor;
 		const hasConfiguration = !!editor.config.get( 'ckbox' );
+		const replaceImageSourceCommand = editor.commands.get( 'replaceImageSource' );
 		const isLibraryLoaded = !!window.CKBox;
 
+		if ( replaceImageSourceCommand ) {
+			this.listenTo<DecoratedMethodEvent<ReplaceImageSourceCommand, 'cleanupImage'>>(
+				replaceImageSourceCommand,
+				'cleanupImage',
+				( _, [ writer, image ] ) => {
+					writer.removeAttribute( 'ckboxImageId', image );
+				}
+			);
+		}
```

Comment:
Shouldn't this code be placed within the `_initConversion()` or `_initFixers()` to keep it closer to its source?